backend/DTOs/UserRegistrationDTO.cs

```diff
@@ -18,19 +18,16 @@ public class UserRegistrationDTO
         public string ConfirmPassword { get; set; }
         [Required]
         public string Country { get; set; }
+        public UserRole Role { get; set; } = UserRole.User;
+
+        [Required]
+        public string PersonalLink { get; set;}
+
         [Required]
-        public UserRole Role { get; set; }
+        public string ProfilePictureLink {get; set;}
 
-        public UserRegistrationDTO() { }
+        [Required]
+        public string Bio { get; set; }
 
-        public UserRegistrationDTO(string email, string firstName, string lastName, string password, string country, UserRole role)
```

Comment:
Why are we removing the constructor?

Reply:
Because we don`t need it ..we dont have any custom logic